examples/dgfip_c/ml_primitif/build.sh

```diff
@@ -87,7 +87,7 @@ else
 
   cd ./calc
 
-  $CCOMPILER -std=c89 -pedantic $CCOPTIONS -O2 -c irdata.c enchain.c var.c contexte.c famille.c revenu.c revcor.c penalite.c variatio.c tableg01.c restitue.c chap-*.c res-ser*.c coc*.c coi*.c horiz*.c
+  $CCOMPILER -std=c89 -pedantic $CCOPTIONS -O2 -c irdata.c enchain.c var.c contexte.c famille.c revenu.c revcor.c penalite.c variatio.c tableg??.c restitue.c chap-*.c res-ser*.c coc*.c coi*.c horiz*.c
```

Comment:
Is the `??` intended or is it bash magic to be replaced by the correct characters later?

Reply:
It's just a wildcard character, like `*`. `?` means "any single character".

Reply:
It's bash magic : one and exactly one character per "?". If k=2, then tableg01.c and tableg02.c are generated, and so on.

Reply:
We never stop learning, thanks :)

Reply:
I beg to differ. This is no magic (it would imply we do not understand what happens). It's a feature ;).